docs/Temperature sensors.md

```diff
@@ -11,6 +11,24 @@ Up to 8 can be connected to the flight controller.
 * Supply: 2.7 to 5.5V
 * Temperature range: -55 to +125°C
 
+On the purple LM75 (CJMCU-75), adress pins on the bottom of the PCB needs to be sorted to get it works (define i2c adress)
+![image](https://user-images.githubusercontent.com/40316777/218416736-b30f364f-aaeb-426e-b211-ea7ef1f3c20e.png)
```

Comment:
If you are referencing an image in the docs, add it to the repo and reference it from there.

You can probably add the image in the assets folder.

Reply:
OK, I will try, but which location in repo is OK to use? do I have rights for that?

Reply:
`https://github.com/iNavFlight/inav/tree/master/docs/assets/images` is a perfect place to store images

Reply:
> OK, I will try, but which location in repo is OK to use? do I have rights for that? edit: seems like I have no right to add it to assets "Uploads are disabled. File uploads require push access to this repository."

You can add it as a new file in your branch, so it is included in the pull request.

Reply:
thanks guys, so I placed picture to my branch, but can't see any note about it in this pull request

Reply:
You forgot to update the reference in the md file.

Now it should read just:

assets/images/218416736-b30f364f-aaeb-426e-b211-ea7ef1f3c20e.png instead of the full https link.

Reply:
Ah, of course, should be OK now